api/box.ts

```diff
@@ -0,0 +1,68 @@
+import { encodeBase64, decodeBase64 } from '../base64';
+import { BoxAcquireRequest, BoxAcquireResponse, BoxProvideRequest } from '../schema';
+import { open, seal } from '../sealedbox';
+
+export async function provideBox<T>(baseUri: string, data: T, publicKey: Uint8Array): Promise<void> {
+  const enc = new TextEncoder();
+
+  // Encode data to JSON, to bytes
+  const rawData = enc.encode(JSON.stringify(data));
+
+  // Seal bytes into box
+  const sealedBox = seal(rawData, publicKey);
+
+  // Base64 parameters into request
+  const body: BoxProvideRequest = {
+    key: await encodeBase64(publicKey),
+    data: await encodeBase64(sealedBox),
+  };
+
+  // Send request
+  const response = await fetch(`${baseUri}/v1/box/provide`, {
```

Comment:
Probably want to set `Content-Type` here and for the request below as well?

Reply:
Sure, don't need to but certainly doesn't hurt.